### PR TITLE
hub and maintainer links to docs.eluv.io 

### PR DESCRIPTION
**Pull request by elv-victor-amigo**

### CODE REVIEW

File: src/main/static/documents/PrivacyPolicy.html
Comment (elv-kevin) [inline src/main/static/documents/PrivacyPolicy.html:1]: why did the html documents get messed up?

**Comment by elv-kevin** [comment]
quick spot check on the links and code search for 'hub.docs.eluv.io' looks good. please remove the changes to the html files and I'll merge and deploy this

File: src/main/static/documents/PrivacyPolicy.html
Comment (elv-victor-amigo) [reply src/main/static/documents/PrivacyPolicy.html:1]: I only added some formatting, no structural changes and got pulled into this request unintentionally

Comment (elv-kevin) [reply src/main/static/documents/PrivacyPolicy.html:1]: Ok, I see. the diff in the files changed section made it look like it was all deleted except for the opening html tag

Comment (elv-victor-amigo) [reply src/main/static/documents/PrivacyPolicy.html:1]: Reverted back, should be back to original

**Comment by elv-kevin** [comment]
sorry @elv-victor-amigo I misunderstood the html diffs. the changes you made there should be fine to include as well.

**Comment by elv-victor-amigo** [comment]
@elv-kevin ok, should be good now

**Pull request merged**